### ISSUE TITLE
Cannot rename/cut any file in AndroidIDE $HOME and its subpath using external app

### ISSUE THREAD

**Issue by BanDroid** [AndroidIDEOfficial/AndroidIDE#1188]
### What happened?

i'm creating project inside $HOME/projects because i need some kind of special execute permission, but i could only rename or cut files/folders using androidide and not my file manager, it gives UnsupportedOperation.

### What's the expected behavior?

could rename or cut files/folders inside $HOME. i have refference for this in termux too, the official termux have the same bug, but someone already fixed it within his own fork.
[reference](https://github.com/termux/termux-app/issues/2415)

### What version of AndroidIDE you're using?

v2.5.0 (debug builds)

### Relevant log output

```shell
-
```

### Duplicate issues

- [X] This issue has not been reported yet.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by BanDroid** [comment]
[here](https://github.com/RebelTat/termux-app/pull/3) is the fork i'm mentioning.

**Comment by itsaky** [comment]
Have you tried that fork? Does it work as expected?

**Comment by BanDroid** [comment]
> Have you tried that fork? Does it work as expected?

yes, i'm using termux from that fork and does work, i could rename files/folders using my file manager, hope androidide could fix it too.
